### ISSUE TITLE
Type acquisition incomplete for Material-UI

### ISSUE THREAD

**Issue by eps1lon** [microsoft/TypeScript-Website#248]
Repro: [importing from `@material-ui/core`](https://www.typescriptlang.org/play/?jsx=2#code/JYWwDg9gTgLgBAKjgQwM5wEoFNkGN4BmUEIcA5FDvmQNwBQokscA3nXHAEICuMMEAOzoBfOERLkAAiGQwsUYMgA2AWm7AA9LmhYydOgW4D8wQXAAiWEBAAUASlbs4lGNygC4AHh58zAN2QFZAEYAF4AIlQACwhuJQATFRgo4gB3cLgAPk8NH34BTJEgA)

The full props of the component are not available. Only `key` is shown in the autocomplete (which originates from the JSX typings as far as I know). Wrong types for props are not flagged as errors.

Console contains many error messages similar to 

`main-3.js:128 Could not get root d.ts file for the module '@material-ui/core' at ./Drawer.d.ts - will not try again in this session 404  Response`.

We have a base `index.d.ts` that re-exports from other declarations file e.g.

`export { default as Drawer }  from './Drawer'` where `Drawer/index.d.ts` contains `export { default } from './Drawer.dts'`.

folder structure
```bash
core/
├── index.d.ts
├── Drawer/
│   ├── index.d.ts
│   ├── Drawer.d.ts
```

On which branch could I work to resolve this issue i.e. is the current version still maintained or all work being done on the v2 branch?

**Comment by orta** [comment]
The current version isn't really being maintained at this point, should hopefully be replaced by the end of the month - current version is https://github.com/microsoft/TypeScript-Website/blob/v2/packages/sandbox/src/typeAcquisition.ts which isn't well tested but at least is documented internally

**Comment by orta** [comment]
Gave this a fresh look as the ATA has had a bunch of upgrades lately - this code sample still raises a lot of errors, there's a lot of files which do resolve but I think the biggest issue is we're accidentally adding a "./" to some of these paths:

![Screen Shot 2021-03-10 at 10 02 24 AM](https://user-images.githubusercontent.com/49038/110612042-b7d9bb80-8187-11eb-8919-1f40d95a8ac5.png)

**Comment by eps1lon** [comment]
@orta Do you have automated tests for ATA that I can contribute to? From there on I might be able to work on fixes for https://github.com/microsoft/TypeScript-Website/issues/248 and https://github.com/microsoft/TypeScript-Website/issues/206. I looked at this in the past but the contribution story was less then ideal since I had to test it on the built website.

**Comment by orta** [comment]
Nope, it relies on a bunch of infra provided by the site to run and hasn't really been worth the time to make a full test suite for me as it rarely changes

**Comment by eps1lon** [comment]
No longer reproduces. Probably fixed. Thanks!

**Comment by orta** [comment]
Yeah, ATA got a re-write with a new algorithm and that probably fixed it up

**Comment by eps1lon** [comment]
Path imports still don't work though (https://github.com/microsoft/TypeScript-Website/issues/206). But considering MUI works now, it might be a lot easier to get path imports to work.
